lib/Sema/CSRanking.cpp

```diff
@@ -97,6 +97,13 @@ static bool shouldIgnoreScoreIncreaseForCodeCompletion(
       return true;
     }
   }
+
+  // If we are completing a member of a literal, consider completion results for
+  // all possible literal types.
+  // E.g. show completion results for `let a: Double = 1.#^COMPLETE^#
+  if (isa_and_nonnull<CodeCompletionExpr>(cs.getParentExpr(expr)) && kind == SK_NonDefaultLiteral) {
```

Comment:
Nit: I think it would be great to move this up into the `if (auto parent = cs.getParentExpr(expr))`

Reply:
Good idea 👍🏽